sql/src/main/java/io/crate/operation/projectors/ShardingUpsertExecutor.java

```diff
@@ -302,6 +303,7 @@ private void countdown() {
                 listener,
                 BACKOFF_POLICY
             );
+            LOGGER.debug("Executing retry Listener for request: {}", request);
```

Comment:
This log statement is not correct. This should be in scheduleConsumeIterator. (or if we really want to log it in the retry loop we should log it in the command we are passing to the retryListener ie:  l -> { 
LOGGER.debug("Executing retry Listener for request: {}", request);
requestExecutor.execute(request, l);
}..., )